provider/aws/service.go

```diff
@@ -44,6 +44,8 @@ func (p *AWSProvider) ServiceCreate(name, kind string, params map[string]string)
 		req, err = createServiceURL(s, "tcp", "tcp+tls", "udp")
 	case "webhook":
 		s.Parameters["Url"] = fmt.Sprintf("http://%s/sns?endpoint=%s", os.Getenv("NOTIFICATION_HOST"), url.QueryEscape(s.Parameters["Url"]))
+		s.Parameters["NotificationTopic"] = notificationTopic
```

Comment:
Go ahead and set these for all service types. There should be some code later that pulls off the unneeded parameters but this way any service that needs them can get them.